tests/simulations/ringtest/create_data.py

```diff
@@ -59,18 +69,13 @@ class SonataAttribute:
 EDGE_TYPES = {attr.name: attr for attr in EDGE_TYPES}
 
 
-def _expand_values(attr, value, count):
+def _expand_values(value, count):
     if isinstance(value, str):
-        ds_value = [value] * count
-    elif isinstance(value, Sequence):
```

Comment:
this is to handle the fact that there were special values required for, `"syn_type_id": [60, 104, 77]`; ie that the second value was > 100 and thus an excitatory synapse.

If you're making the ring variable, IMO, it's less clear where the excitatory synapses should be.  Right now they would be `Inh, Exc, Inh, Inh`, in the case of PopA if the count is increased to 4, and `Inh, Inh, Inh` for PopB.

Reply:
I do not see the problem. You write I, E, I and I am going to cycle over that as much as I need to. What is unclear? What would you envision instead?

Reply:
> You write I, E, I and I am going to cycle over that as much as I need to. What is unclear?

It's unclear in the sense that the total length is determined by the count, but in a completely different place a position sensitive list exists; hence the check on the length to minimize errors.

> Also, notice that if you provide the correct-size vector it will place the values in the correct places, as before.

Yes, however, it's length is not checked, and that's an easy mistake to make.

> It is not feasible to write by hand an array of 60k elements

Something like this works: `[29, 129, 29]*60_000`, or you can use something from iterator; it's just that it's explicit in that case.